include/nc4internal.h

```diff
@@ -205,6 +205,7 @@ typedef struct NC_VAR_INFO
     void *fill_value;
     size_t *chunksizes;
     nc_bool_t contiguous;        /**< True if variable is stored contiguously in HDF5 file */
+    nc_bool_t compact;           /**< True if variable is in comact storage in HDF5 file */
```

Comment:
I'm not sure if this matters in C, but if this were C++, it would save space in the struct (due to alignment concerns) to have all the `nc_bool_t` together instead of having the `int parallel_access` in between.  I haven't measured whether it makes any difference in C.

Reply:
It looks like the `nc_bool_t` size is 4 bytes which is the same size as an int, so intermingling int and nc_bool_t doesn't change the size of the struct in this case.  If the `nc_bool_t` were changed to the stdbool-defined `bool`, the size of the struct would drop by 40-bytes (something for the future...)

Reply:
nc_bool_t is just an int. If it ever changes, then whoever makes the change will be in charge of making the correct change. I don't generally code based on this kind of thinking. I can only code correctly, and hope that future netCDF programmers will do the same. ;-)